analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt

```diff
@@ -38,7 +38,7 @@ class FilePackageCurationProviderConfig(
     /**
      * The path of the package curation file or directory.
      */
-    val path: String
+    val path: File
```

Comment:
The property should also be renamed to `path`.
Commit message: "simplifies"

Reply:
> The property should also be renamed to `path`.

Uhm, the property is already named `path`...

> Commit message: "simplifies"

:+1:

Reply:
> Uhm, the property is already named `path`...

But if you actually meant "file", then I'd prefer to stick with "path" for the same reason as mentioned [here](https://github.com/oss-review-toolkit/ort/pull/6326#discussion_r1067125867), and to not break the config again.

Reply:
Yes, I meant `file`.